int16 visible
int16 xoffset
int16 yoffset
int16 img_width
int16 img_height
